src/app/components/Application/Main.jsx

```diff
@@ -192,6 +194,7 @@ class Main extends React.Component {
           currentAudience={this.props.currentAudience}
           displayInfo={this.getPicksInfo(this.props.picksData, this.props.currentAudience)}
           picksCount={picksCount}
+          type={this.props.picksData.type}
```

Comment:
Change to displayType. Fixes issue with Staff Picks titles.

Reply:
What does your comment mean? Also, on line 179 `type` is defined in `const { type } = this.props.picksData;`. That's the same (but different syntax) for `const type = this.props.picksData.type;`

Reply:
I'm confusing my variables here. Looks like I need to just pick a name and use that consistently. Was thinking of using `displayType` to use a similar construct to `displayInfo` but should just use type instead. I'll remove the extra assignment at line 179 -- leftover from testing variable outputs, I believe.

Reply:
Cool, leaving the variable there and just using `type={type}` would've been okay, too.